tests/unit/ceed/test_ceed.cpp

```diff
@@ -427,14 +514,35 @@ TEST_CASE("CEED mass & diffusion", "[CEED]")
    auto pb = GENERATE(Problem::Mass,Problem::Diffusion,Problem::MassDiffusion,
                       Problem::VectorMass,Problem::VectorDiffusion);
    auto order = GENERATE(1);
-   auto mesh = GENERATE("../../data/inline-quad.mesh","../../data/inline-hex.mesh",
+   auto mesh = GENERATE("../../data/inline-quad.mesh",
```

Comment:
I think this is the first time I have seen the Catch generators in action. I like it quite a bit.